src/containers/Footer/index.jsx

```diff
@@ -229,7 +114,6 @@ const Footer = (props) => {
 }
 
 Footer.propTypes = {
-  language: PropTypes.string.isRequired,
   actions: PropTypes.shape({
     updateLanguage: PropTypes.func,
```

Comment:
You can remove updateLanguage too which will allow the removal of redux and props from this component completely